pkg/runtime/runtime.go

```diff
@@ -1418,6 +1422,33 @@ func (a *DaprRuntime) initLock(s components_v1alpha1.Component) error {
 	return nil
 }
 
+func (a *DaprRuntime) initWorkflow(s components_v1alpha1.Component) error {
+	// create the component
+	workflowComp, err := a.workflowsRegistry.Create(s.Spec.Type, s.Spec.Version)
+	if err != nil {
+		log.Warnf("error creating workflow component %s (%s/%s): %s", s.ObjectMeta.Name, s.Spec.Type, s.Spec.Version, err)
+		diag.DefaultMonitoring.ComponentInitFailed(s.Spec.Type, "creation")
+		return err
+	}
+	if workflowComp == nil {
+		return nil
+	}
+	// initialization
+	props := a.convertMetadataItemsToProperties(s.Spec.Metadata)
+	err = workflowComp.Init(wfs.Metadata{
+		Properties: props,
+	})
+	if err != nil {
+		diag.DefaultMonitoring.ComponentInitFailed(s.Spec.Type, "init")
+		log.Warnf("error initializing workflow component %s (%s/%s): %s", s.ObjectMeta.Name, s.Spec.Type, s.Spec.Version, err)
```

Comment:
Same comment about using `%v` for errors
```suggestion
		log.Warnf("error initializing workflow component %s (%s/%s): %v", s.ObjectMeta.Name, s.Spec.Type, s.Spec.Version, err)
```